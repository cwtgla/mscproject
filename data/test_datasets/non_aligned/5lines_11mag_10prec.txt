2047.1023
113.0913
0.1011
-993.0011
-2047.1021