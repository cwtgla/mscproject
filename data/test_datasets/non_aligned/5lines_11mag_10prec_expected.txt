127
255
252
28
121
16
0
252
239
132
11
255
255
244